plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/EverytradeCSVParserValidator.java

```diff
@@ -0,0 +1,51 @@
+package io.everytrade.server.plugin.impl.everytrade.parser;
+
+import io.everytrade.server.plugin.api.parser.TransactionCluster;
+import io.everytrade.server.plugin.impl.everytrade.parser.exchange.ExchangeBean;
+import io.everytrade.server.util.CurrencyUtil;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class EverytradeCSVParserValidator {
+
+    public static Map<String,String> parsedSymbol(String value) {
+        Map<String,String> symbols = new HashMap<>();
+        String[] symbolParts = symbolPairParser(value);
+        String base = correctCurrency(symbolParts[0].toUpperCase());
```

Comment:
here you call `toUpperCase` but in  `correctCurrency` method all you do is `toUpperCase`. So here it is redundant